integration/src/main/kotlin/kotlinx/benchmark/integration/GradleTestVersion.kt

```diff
@@ -1,6 +1,7 @@
 package kotlinx.benchmark.integration
 
 enum class GradleTestVersion(val versionString: String) {
+    v7("7.6.4"),
```

Comment:
I guess this is redundant now?

Reply:
```suggestion
```

yup, sure looks like it, I'll remove it.